phip/cli.py

```diff
@@ -187,6 +196,117 @@ def clipped_factorization_model(
     print("Wrote: %s" % output_path)
 
 
+@cli.command(name='call-hits')
+@option('-i', '--input', required=True, type=Path(exists=True, dir_okay=False),
+    help='input counts file (tab-delim)')
+@option('-o', '--output', required=False, type=Path(exists=False),
+    help='output file or directory. If ends in .tsv, will be treated as file')
+@option('-d', '--index-cols', default=1,
+    help='number of columns to use as index/row-key')
+@option('--beads-regex', default=".*beads.*", show_default=True,
+    help='samples with names matching this regex are considered beads-only')
+@option('--ignore-columns-regex', default="^_background.*", show_default=True,
+    help='ignore columns matching the given regex. Ignored columns are passed '
+    'through to output without processing.')
+@option('--ignore-rows-regex', default="^_background.*", show_default=True,
+    help='ignore rows matching the given regex')
```

Comment:
I would note on these comments that the regex is processed case-insensitively

Reply:
👍